.github/workflows/tests.yml

```diff
@@ -19,22 +19,52 @@ jobs:
 
     steps:
     - uses: actions/checkout@v1
+
     - name: Set up Python 3.6
       uses: actions/setup-python@v1
       with:
         python-version: 3.6
+
+    - name: Pip Cache
+      uses: actions/cache@v1
+      id: pip-cache
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('Pipfile.lock') }}
```

Comment:
I wonder if you should also has `Pipfile`, if for example you change some setting or something like that?

Reply:
I don't _think_ that's necessary, but I'm not entirely sure. Right now I'm mostly experimenting to see if I can get this `cache` action to work as I want, hence all the commits. That action is a bit weird and needs to be specified once per directory that needs caching.

Reply:
Right, no, it shouldn't be necessary to put the `Pipfile` in the hash as well, since the lock file will change whenever you make changes to the Pipfile, even if you change settings for the sources or such.